configure.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+echo "Running configuration script"
+export DEBIAN_FRONTEND=noninteractive
+
+echo "US/Central" | sudo tee /etc/timezone
+dpkg-reconfigure --frontend noninteractive tzdata
+
+echo "Install essential software pacakges"
+apt-get -qq update
+apt-get -qq install -y wget htop wget build-essential zip software-properties-common gnupg awscli
+apt-get -qq install -y ansible
+echo $PERM_ENV  > /data/www/host.txt
+
+echo "Add custom sources"
+# Add mysql key
+sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 5072E1F5
+# Add psql key
+sudo sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list'
```

Comment:
Why are we installing... keys... related to PostgreSQL... help?  There is obviously some practice here that I'm not familiar with.

Reply:
(See my comment to the mysql version of this; taking these out)